### ISSUE TITLE
Kerion

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9987]
**Preferred term label:**
Kerion

**Synonyms**
Erythematous, boggy, tender, often sterile, weeping nodule or plaque with pustules and draining sinuses

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Tinea barbae

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal
I think that Kerion is probably better characterized as a disease entity?
https://pubmed.ncbi.nlm.nih.gov/27696388/

**Comment by MickeySegal** [comment]
Here it is presented as a characteristic lesion of a condition:
https://www.ncbi.nlm.nih.gov/books/NBK563204/
"The classic inflammatory form of tinea barbae develops a characteristic lesion called kerion. Kerion is an erythematous, boggy, tender, often sterile, weeping nodule or plaque with pustules and draining sinuses."

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000424
- label: Kerion
- parents: [Localized skin lesion (HP:0011355)](https://hpo.jax.org/app/browse/term/HP:0011355)
- pmid: PMID:33085351
- def: Kerion is an erythematous, boggy, tender, often sterile, weeping nodule or plaque with pustules and draining sinuses. Hair in these areas appears lusterless, brittle, and is easily plucked. Kerion can be solitary or multiple but is usually unilateral. Patients with inflammatory tinea barbae may have constitutional symptoms like fever and malaise. Kerion is a coalesced sterile pustule, but occasionally they can get superinfected with cutaneous bacteria and develop regional lymphadenopathy.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9987
- synonyms:
